src/Shared/SearchWrapper.js

```diff
@@ -0,0 +1,192 @@
+import React, { Component } from 'react'
+import {
+  View,
+  Text,
+  StyleSheet,
+  Image,
+  TextInput,
+  Platform,
+} from 'react-native'
+import Icon from 'react-native-vector-icons/dist/MaterialIcons'
+
+export default class SearchBarWrapper extends Component {
+  defaultProps = {
+    placeholderText: 'Search...',
```

Comment:
I think the reason defaultProps didn't work as you expected here is that you need to place these keys inside the `searchBar` object. Try doing that, or just removing this code.